specifyweb/specify/tree_extras.py

```diff
@@ -299,6 +299,21 @@ def merge(node, into, agent):
 
     assert False, "failed to move all referrences to merged tree node"
 
+def bulk_move(node, into, agent):
+    from . import models
+    logger.info('Bulk move preparations children %s into %s', node, into)
+    model = type(node)
+    if not type(into) is model: raise AssertionError(
+        f"Unexpected type of node '{into.__class__.__name__}', during bulk move. Expected '{model.__class__.__name__}'",
+        {"node" : into.__class__.__name__,
+        "nodeModel" : model.__class__.__name__,
+        "operation" : "bulkMove",
+        "localizationKey" : "invalidNodeType"})
+    target = model.objects.select_for_update().get(id=into.id)
+    if not (node.definition_id == target.definition_id): raise AssertionError("bulk move across trees", {"localizationKey" : "bulkMoveAcrossTrees"})
```

Comment:
The `bulkMoveAcrossTrees` localization key is not resolved or recognized on the frontend. 

You will likely need to add the frontend implementation. 
See how similar tree actions are being resolved: 
 
https://github.com/specify/specify7/blob/b331ce9c552e90d86fc4811a7089fab6ac29c07e/specifyweb/frontend/js_src/lib/components/Errors/JsonError.tsx#L180-L183

Since these three error message are very similar, consider making a more generic `{operation} across trees` and remove the more specific error messages in favor of the generic one. 

Alongside the `{operation} across trees` message, you will need to pass in the operation name in the payload being sent with the error. 

For a similar example, see https://github.com/specify/specify7/blob/b331ce9c552e90d86fc4811a7089fab6ac29c07e/specifyweb/specify/tree_extras.py#L245-L248